src/js/popup/popup.js

```diff
@@ -1,802 +1,1350 @@
+              
+              const maskListItemAddressBar = document.createElement("div");
+              maskListItemAddressBar.classList.add("fx-relay-mask-item-address-bar");
+
+              const maskListItemAddressWrapper = document.createElement("div");
+              maskListItemAddressWrapper.classList.add("fx-relay-mask-item-address-wrapper");
+
+              const maskListItemLabel = document.createElement("span");
+              maskListItemLabel.classList.add("fx-relay-mask-item-label");
+              maskListItemLabel.textContent = mask.description;
+              
+              // Append Label if it exists
+              if (mask.description !== "") {
+                maskListItemAddressWrapper.appendChild(maskListItemLabel);
+              }
+              
+              const maskListItemAddress = document.createElement("div");
+              maskListItemAddress.classList.add("fx-relay-mask-item-address");
+              maskListItemAddress.textContent = mask.full_address;
+              maskListItemAddressWrapper.appendChild(maskListItemAddress);
+
+              // Add Mask Address Bar Contents 
+              maskListItemAddressBar.appendChild(maskListItemAddressWrapper);
+
+              const maskListItemAddressActions = document.createElement("div");
+              maskListItemAddressActions.classList.add("fx-relay-mask-item-address-actions");
+
+              const maskListItemCopyButton = document.createElement("button");
+              maskListItemCopyButton.classList.add("fx-relay-mask-item-address-copy");
+              maskListItemCopyButton.setAttribute("data-mask-address", mask.full_address);
+
+              const maskListItemCopyButtonSuccessMessage = document.createElement("span");
+              maskListItemCopyButtonSuccessMessage.textContent = browser.i18n.getMessage("popupCopyMaskButtonCopied");
+              maskListItemCopyButtonSuccessMessage.classList.add("fx-relay-mask-item-address-copy-success");
+              maskListItemAddressActions.appendChild(maskListItemCopyButtonSuccessMessage);
+              
+              maskListItemCopyButton.addEventListener("click", (e)=> {
+                e.preventDefault();
+                navigator.clipboard.writeText(e.target.dataset.maskAddress);
+                maskListItemCopyButtonSuccessMessage.classList.add("is-shown");
+                setTimeout(() => {
+                  maskListItemCopyButtonSuccessMessage.classList.remove("is-shown")
+                }, 1000);
+              }, false);
+              maskListItemAddressActions.appendChild(maskListItemCopyButton);
+
+              const maskListItemToggleButton = document.createElement("button");
+              maskListItemToggleButton.classList.add("fx-relay-mask-item-address-toggle");
+              maskListItemToggleButton.addEventListener("click", ()=> {
+                // TODO: Add Toggle Function
+              }, false);
```

Comment:
Suggestion: Should we remove and create an issue for this?

Reply:
[This one](https://github.com/mozilla/fx-private-relay-add-on/pull/473) is also your's! 😉